### PR TITLE
Add dockerfile for build centos image

### PR DESCRIPTION
**Pull request by RidmiR**
## Purpose
Purpose of this PR is to create a docker image form centos base image to create centos installers.
Change the folder structure to handling multiple dockerfiles.   

## Approach
Created dockerfile by testing it locally, identified the relevant packages that needs to install.

### CODE REVIEW

File: dockerfiles/centos/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# license, please see the license as well as any agreement you’ve entered into
+# with WSO2 governing the purchase of this software and any associated services.
+# ------------------------------------------------------------------------------
+
+# set base Docker image to centos7.7.1908 CentOS Docker image
+FROM centos:centos7.7.1908
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
+
+# install required packages
+RUN yum -y update && yum install -y rpm-build sudo git unzip wget
```
Comment (chirangaalwis) [inline dockerfiles/centos/Dockerfile]: @RidmiR let us go with multi lines when defining package names and other best practices. Please see the format we have used in official product Docker source repositories (e.g. [APIM](https://github.com/wso2/docker-apim/blob/v3.1.0.3/dockerfiles/centos/apim/Dockerfile))

Comment (RidmiR) [reply dockerfiles/centos/Dockerfile]: @chirangaalwis Intially I wrote the docker file with multiple lines and it throws Warning. Show in Below

 **_[WARNING]: Empty continuation line found in:
    RUN yum -y update && yum install -y rpm-build sudo git unzip wget RUN echo "net.ipv4.ip_local_port_range=15000 61000" >> /etc/sysctl.conf && echo "net.ipv4.tcp_fin_timeout=30" >> /etc/sysctl.conf && echo "*	soft	nofile	65535" >> /etc/security/limits.conf && echo "*	hard	nofile	65535" >> /etc/security/limits.conf && echo "*	soft	nproc	65535" >> /etc/security/limits.conf && echo "*	hard	nproc	65535" >> /etc/security/limits.conf_**

As a fix, I remove and make it one line. Do I need to revert it?

File: Dockerfile [hunk 2]
```diff
@@ -222,4 +222,4 @@ COPY entrypoint.sh /usr/local/bin/entrypoint.sh
 
 ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
 
-CMD ["/usr/sbin/sshd", "-D"]
+CMD ["/usr/sbin/sshd", "-D"]
```
Comment (chirangaalwis) [inline Dockerfile]: @RidmiR please add an EOF line.

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: dockerfiles/centos/Dockerfile (hunk 1 above)
Comment (RidmiR) [reply dockerfiles/centos/Dockerfile]: Fixed by changing into multiple lines

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**